agent/src/agent.ts

```diff
@@ -56,26 +55,27 @@ export const agent = new Agent({
         key: AGENT_WALLET_KEY,
       },
     }),
-    openId4VcIssuer: new OpenId4VcIssuerModule({
-      baseUrl: joinUriParts(AGENT_HOST, ['oid4vci']),
-      router: openId4VciRouter,
-      credentialRequestToCredentialMapper,
-      deferredCredentialRequestToCredentialMapper,
-      getVerificationSessionForIssuanceSessionAuthorization: getVerificationSessionForIssuanceSession,
-    }),
-    openId4VcHolder: new OpenId4VcHolderModule(),
-    openId4VcVerifier: new OpenId4VcVerifierModule({
-      baseUrl: joinUriParts(AGENT_HOST, ['oid4vp']),
-      router: openId4VpRouter,
-      authorizationRequestExpirationInSeconds: 600,
-      federation: {
-        async getAuthorityHints(agentContext, { verifierId }) {
-          return getAuthorityHints(verifierTrustChains, verifierId).map((verifierId) =>
-            joinUriParts(AGENT_HOST, ['oid4vp', verifierId])
-          )
-        },
-        async isSubordinateEntity(agentContext, options) {
-          return isSubordinateTo(verifierTrustChains, options.verifierId, options.subjectEntityId).length > 0
+    oid4vc: new OpenId4VcModule({
```

Comment:
if you use openid4vc you can access it as `agent.openid4vc`

```suggestion
    openid4vc: new OpenId4VcModule({
```